packages/app-page-builder/src/editor/plugins/elements/button/ButtonContainer.tsx

```diff
@@ -20,10 +20,9 @@ const excludePlugins = [
 const ButtonContainer = props => {
     const { getAllClasses, elementStyle, elementAttributes, element } = props;
     const { type = "default", icon = {} } = element.data || {};
-    const svg = icon.svg || null;
     const { alignItems } = elementStyle;
 
-    const { position = "left" } = icon;
+    const { svg = null, position = "left" } = icon || {};
```

Comment:
I think we can reduce `icon || {}` to just `icon` because we're already defining the default value for `icon` above in line number 22.